easybuild/easyconfigs/e/enable/enable-20160817_numpy.patch

```diff
@@ -0,0 +1,16 @@
+#We do not have a deafault numpy-site.cfg, so numpy searches X11 libs only at standard locations.
+#Here, we direclty add the location of the EB provided libX11.
+# Aug 25th 2016 by B. Hajgato (Free University Brussels - VUB)
+--- enable-168e222dababd4464062eb7a3ab92c4b591f7031/kiva/agg/setup.py.org	2016-08-17 17:12:22.000000000 +0200
++++ enable-168e222dababd4464062eb7a3ab92c4b591f7031/kiva/agg/setup.py	2016-08-25 11:31:29.272582528 +0200
+@@ -235,7 +235,9 @@
+         # Frequently, the 64-bit libraries are not in a known location and need
+         # manual configuration. From experience, this is usually not detected by
+         # the builder if we do not raise an exception.
+-        x11_info = get_info('x11', notfound_action=2)
++        x11_info = {'libraries': ['X11'],
++                    'library_dirs': [os.path.join(os.environ["EBROOTLIBX11"], "lib")],
++                    'include_dirs': [os.path.join(os.environ["EBROOTLIBX11"], "include")]}
```

Comment:
hmm, ok, good point @hajgato, it would be a lot of `$EBROOT`'s, indeed... and I guess it's fairly rare to have to grab the install prefix for one of the components